frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
```

Comment:
Prefer TypeScript for new components.

Reply:
Sure, just an observation for future work. I won't ask you to convert all this :)